clusters/app.ci/prow-monitoring/prow_alertmanager.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   containers:
   - args:
     - -provider=openshift
-    - -https-address=:9094
+    - -https-address=:9095
```

Comment:
Alertmanager started to use 9094 in the meantime, resulting in a bind error and hence crashloop